### PR TITLE
fix: fix a CI parameter typo

### PR DESCRIPTION
**Pull request by hamidzr**
## Checklist

- [x] User-facing API changes need the "User-facing API Change" label.
- [x] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

**Review by stoksc — APPROVED** [review]

**Comment by hamidzr** [comment]
https://github.com/determined-ai/determined/pull/1472 should fix the cli failing waiting on that before we land this

**Comment by hamidzr** [comment]
closing in favor of https://github.com/determined-ai/determined/pull/1473 so we can run the full test suite

**Pull request closed without merging**
